plugins/shared/info-panel/style.less

```diff
@@ -172,10 +172,13 @@
         &-description {
             font-size: @descriptionFontSize;
             padding: 10px 0 0;
+            user-select: text;
```

Comment:
The description is inheriting the `tota11y-no-select` class styles, so this is to override that and make the text selectable. I'm not super familiar with Less yet so there may be a better way to do this.